### PR TITLE
Updating the Edit Players page to be clear about what fields can be used

### PR DESCRIPTION
**Pull request by Swimminschrage**
When I was playing around with KFC, I found it confusing what the exact fields should be and what other formats were available for creating my party.  Just wanted to submit a quick PR offering to clarify some of the fields and to add some ghost text to give examples.

### CODE REVIEW

File: app/players/edit.html [hunk 1]
```diff
@@ -1,9 +1,14 @@
 <div class="edit-players">
-	<p>One character per line. Blank line to separate different parties. Format: name initiative hp</p>
+	<p>One character per line. Blank line to separate different parties. Format:<br/>
```
Comment (Asmor) [inline app/players/edit.html]: Please move the first line to its own line (i.e. &lt;p&gt; should be on a line by itself) and indent the contents of the &lt;p&gt; tag.

Thanks!

**Comment by Asmor** [comment]
One minor formatting change required. See my comment on the file.

I have to warn you though that due to some changes in my development environment I'm currently unable to build KFC, and I can't accept any pull requests until I get that figured out. I haven't had time to solve it yet and I'm not sure when I will, but I will merge outstanding pull requests when I can.

**Comment by Swimminschrage** [comment]
@Asmor Took care of the formatting.  No worries about the deployment.  Let me know if there's anything I can do to help you out.

**Comment by Asmor** [comment]
This is now merged. Thanks!

**Pull request merged**